crates/core/src/models/call.rs

```diff
@@ -0,0 +1,328 @@
+            "0x00",
+            "0x075",
+            "0x075",
+            "0x02",
+            "0x0f8",
+            "0x072",
+            "0x084",
+            "0x04b",
+            "0x04b",
+            "0x052",
+            "0x054",
+            "0x082",
+            "0x0de",
+            "0x0ad",
+            "0x082",
+            "0x0de",
+            "0x0ad",
+            "0x082",
+            "0x0de",
+            "0x0ad",
+            "0x084",
+            "0x03b",
+            "0x09a",
+            "0x0ca",
+            "0x00",
+            "0x094",
+            "0x02e",
+            "0x011",
+            "0x0ed",
+            "0x082",
+            "0x0f5",
+            "0x0ec",
+            "0x016",
+            "0x05a",
+            "0x0b8",
+            "0x0ce",
+            "0x03c",
+            "0x0c0",
+            "0x094",
+            "0x0f0",
+            "0x025",
+            "0x0fe",
+            "0x075",
+            "0x027",
+            "0x0f4",
+            "0x0d1",
+            "0x080",
+            "0x084",
+            "0x0b3",
+            "0x0bc",
+            "0x0fa",
+            "0x082",
+            "0x0c0",
+            "0x01",
+            "0x0a0",
+            "0x088",
+            "0x09b",
+            "0x0e6",
+            "0x07d",
+            "0x059",
+            "0x0bc",
+            "0x01a",
+            "0x043",
+            "0x0dd",
+            "0x080",
+            "0x039",
+            "0x055",
+            "0x0f7",
+            "0x091",
+            "0x07d",
+            "0x0dc",
+            "0x0b7",
+            "0x0d7",
+            "0x048",
+            "0x0ed",
+            "0x03e",
+            "0x09b",
+            "0x00",
+            "0x0cd",
+            "0x0b1",
+            "0x059",
+            "0x0f2",
+            "0x094",
+            "0x065",
+            "0x019",
+            "0x076",
+            "0x0b8",
+            "0x0a0",
+            "0x038",
+            "0x01",
+            "0x070",
+            "0x02a",
+            "0x060",
+            "0x06f",
+            "0x0fb",
+            "0x0fd",
+            "0x060",
+            "0x036",
+            "0x04f",
+            "0x0f8",
+            "0x097",
+            "0x0f7",
+            "0x0ca",
+            "0x051",
+            "0x014",
+            "0x011",
+            "0x0d6",
+            "0x066",
+            "0x0f",
+            "0x093",
+            "0x06d",
+            "0x0d5",
+            "0x01e",
+            "0x0b9",
+            "0x0a",
+            "0x07d",
+            "0x030",
+            "0x073",
+            "0x052",
+            "0x061",
+        ];
+        to_vec_field_element(calldata)
+    }
+
+    #[test]
+    fn test_slice_calls() {
+        // Given
+        let calls: Calls = get_simple_test_calldata().try_into().unwrap();
+
+        // When
+        let slice = calls.get(1..3).unwrap();
+
+        // Then
+        assert_eq!(slice.len(), 2);
```

Comment:
if we had human readable calls as input and not raw bytes, we could assert_eq against a `Call`

Reply:
same comment as above, can move to a human readable json